hedera-node/pom.xml

```diff
@@ -282,6 +282,7 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-transport-native-epoll</artifactId>
+      <classifier>linux-x86_64</classifier>
```

Comment:
From the issue, it seems that we have different environment settings (difference in OS) on some nodes of `previewnet`. Should we have another `classifier` for our current environment, or this should cover both?

Reply:
My impression is that Netty only fails when we [set](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/grpc/ConfigDrivenNettyFactory.java#L91),
```
				.channelType(EpollServerSocketChannel.class)
```
And if we don't set this, then the `-native-epoll` artifact is simply ignored at run-time.

Reply:
@tinker-michaelj, I understand that part. But we have regression tests that force the code that you mentioned to run, and yet this code fails in previewnet, due to different OS, as far as I can tell, hence my comment/question.

Reply:
Are we _sure_ there were regression tests running with `netty.mode=PROD`? 

In any case, when you add this `classifier`, you can see the missing library on the classpath:
```
$ jar tvf hedera-node/data/lib/netty-transport-native-epoll-4.1.66.Final-linux-x86_64.jar | grep 'x86_64.so'
 89445 Fri Jul 16 06:37:54 CDT 2021 META-INF/native/libnetty_transport_native_epoll_x86_64.so
```

So imo we can treat this as the fix until we have further evidence.

Reply:
Agreed @tinker-michaelj. Created #2015 to help catch similar issue earlier.